packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -59,9 +59,9 @@ export const messages = {
     "The forceide:// URL is invalid. From your subscriber's org, copy and paste the forceide:// URL shown on the Apex Debugger page in Setup.",
 
   force_org_create_default_scratch_org_text:
-    'SFDX: Create a Default Scartch Org...',
+    'SFDX: Create a Default Scratch Org...',
 
-  force_org_open_default_scratch_org_text: 'SFDX: Open Default Org',
+  force_org_open_default_scratch_org_text: 'SFDX: Open Default Scratch Org',
```

Comment:
This should not contain `Scratch`. Any type of org can be your default org.

Reply:
Okay!